.github/workflows/release-layer-nodejs.yml

```diff
@@ -0,0 +1,76 @@
+name: "Release NodeJS Lambda Layer"
+
+on:
+  # (Using tag push instead of release to allow filtering by tag prefix.)
+  push:
+    tags:
+      - layer-nodejs/**
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build-layer:
+    runs-on: ubuntu-latest
+    outputs:
+      NODEJS_VERSION: ${{ steps.save-node-sdk-version.outputs.SDK_VERSION}}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+
+      - name: Build
+        run: |
+          npm install
+        working-directory: nodejs
+
+      - name: Save Node SDK Version
+        id: save-node-sdk-version
+        run: |
+          SDK_VERSION=$(node version.js)
+          echo "SDK_VERSION=$SDK_VERSION" >> $GITHUB_OUTPUT
+        working-directory: nodejs/packages/layer/scripts
+
+      - name: Rename zip file
+        run: mv layer.zip opentelemetry-nodejs-layer.zip
+        working-directory: nodejs/packages/layer/build
```

Comment:
I don't think this step is necessary.  Just rename the upload artifact path below to `layer.zip`

Reply:
If I don't rename it, then the downloadable artifact of the completed GH action job will be called `layer.zip`. I was trying to make it consistent with the Java action.

Reply:
Really?  I thought the name was set by the upload artifact.  🤷

Reply:
It does, but the artifact name and the zip file name must be the same with how the publish workflow is written right now. We are passing the artifact name to the publish workflow, and the name is used to 1) download the artifact and 2) as the name of the zip file to upload to AWS. If I use different values, then when you download the artifact in the workflow, you will get the `layer.zip` file (not `[artifact-name].zip`).